### PR TITLE
Redd 608

### PR DESCRIPTION
**Pull request by SubatomicHero**

### CODE REVIEW

File: recipes/_alfrescoproperties-attributes.rb [hunk 1]
```diff
@@ -35,22 +35,19 @@
...
-node.default['artifacts']['ssl-db-creds']['url'] = "http://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem"
-node.default['artifacts']['ssl-db-creds']['destination'] = "/tmp/rds-combined-ca-bundle.pem"
-node.default['artifacts']['ssl-db-creds']['owner'] = "tomcat"
+node.default['artifacts']['ssl-db-creds']['url'] = 'http://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem'
+node.default['artifacts']['ssl-db-creds']['destination'] = '/tmp/rds-combined-ca-bundle.pem'
+node.default['artifacts']['ssl-db-creds']['owner'] = 'tomcat'
 node.default['artifacts']['ssl-db-creds']['enabled'] = false
 
-# HTTP default pool size
-node.default['alfresco']['properties']['httpclient.max-connections'] = "20"
```
Comment (enzor) [inline recipes/_alfrescoproperties-attributes.rb]: why did you delete this?

Comment (SubatomicHero) [inline recipes/_alfrescoproperties-attributes.rb]: This hasn't been deleted but moved with the S3 attributes, as @steverigby mentioned that this is only relevant for the S3 connector.

Comment (enzor) [inline recipes/_alfrescoproperties-attributes.rb]: great, thanks for explaining
